variables.tf

```diff
@@ -76,3 +76,27 @@ variable "tags" {
   type        = map(string)
   description = "A mapping of tags to assign to the bucket"
 }
+
+variable "repl_iam_role_arn" {
+  type = string
+  default = null
+  description = "The ARN of the role that is used for the replication"
+}
+
+variable "repl_dest_bucket" {
+  type = string
+  default = null
+  description = "The destination bucket for the replication"
+}
+
+variable "repl_storage_class" {
+  type = string
+  default = null
+  description = "The storage class for the replication"
+}
+
+variable "lifecycle_rule" {
+  description = "List of maps containing configuration of object lifecycle management."
+  type        = any
+  default     = []
+}
```

Comment:
Can this be updated please, the params should be in the order of `type`, `default`, `description` like other variables.

Could you also run `terraform fmt` against this to fix the formatting.

Reply:
Formatting has been run. There is no convention for ordering yet, let's wait until we're there :)

Reply:
There definitely is a convention for the ordering within our MCAF modules. Just look at all the `variables.tf` and `outputs.tf` files of each module (except the transfer module you created yourself). As you can clearly see they are all consistent, so please order the fields by `type`, `default` (if applicable) and then `description`.

Additionally there is also a specific order in the variable blocks themselves. The `tags` variable is always at the bottom of the file (don't ask). Any variables that uniquely define the main item in the module (`name`, or `description`) are always on top, and the rest of the variables are ordered alphabetically in between.

So these new variables should be ordered a little higher in the file...